hack/common.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
 # Location of the manifests file
-MANIFEST_LOC=bundle/
+if [ "$COMMUNITY" = "true" ]; then
```

Comment:
CI is failing because this is triggering `set -o nounset`
Try doing `if [ -n "$COMMUNITY"  && "$COMMUNITY" = "true" ]; then`

Reply:
Thanks, I was just about to ask about this